.bonsai.yml

```diff
@@ -0,0 +1,58 @@
+---
+description: "#{repo}"
+builds:
+- platform: "linux"
+  arch: "amd64"
+  asset_filename: "#{repo}_#{version}_linux_amd64.tar.gz"
+  sha_filename: "#{repo}_#{version}_sha512-checksums.txt"
+  filter:
+  -  "entity.system.os == 'linux'"
+  -  "entity.system.arch == 'amd64'"
+
+- platform: "linux"
+  arch: "386"
+  asset_filename: "#{repo}_#{version}_linux_386.tar.gz"
+  sha_filename: "#{repo}_#{version}_sha512-checksums.txt"
+  filter:
+  -  "entity.system.os == 'linux'"
+  -  "entity.system.arch == '386'"
+
+- platform: "linux"
+  arch: "arm64"
+  asset_filename: "#{repo}_#{version}_linux_arm64.tar.gz"
+  sha_filename: "#{repo}_#{version}_sha512-checksums.txt"
+  filter:
+  -  "entity.system.os == 'linux'"
+  -  "entity.system.arch == 'arm64'"
+
+- platform: "linux"
+  arch: "armv7"
+  asset_filename: "#{repo}_#{version}_linux_armv7.tar.gz"
+  sha_filename: "#{repo}_#{version}_sha512-checksums.txt"
+  filter:
+  -  "entity.system.os == 'linux'"
+  -  "entity.system.arch == 'armv7'"
```

Comment:
The GOARCH value will only report "arm", so this won't work. Unfortunately the arm version is set by the GOARM environment variable at build time.

Reply:
@echlebek Yeah, I think I understand the problem. I inherited the bonsai file from [sensu-go-plugin](https://github.com/sensu/sensu-go-plugin/blob/master/.bonsai.yml), so I'm not exactly sure how everything works behind the curtains. However, this is perhaps something we'll need to figure out in sensu-go-plugin first, and then apply the fix to all the supported go plugins.